### DISCUSSION TITLE
A patch that works on Dexed but produces no sound on MiniDexed

### DISCUSSION THREAD

**Discussion by thatmichaelpark** [probonopd/MiniDexed#298] [Q&A] [answered]
I've tried to attach a zipped .syx here, not sure it worked: [001_test.syx.zip](https://github.com/probonopd/MiniDexed/files/9026498/001_test.syx.zip)

In that .syx is a patch called Oxonal. It works fine on desktop Dexed but on MiniDexed it produces nothing. I'm hoping someone can explain why.

**Comment by dcoredump** [comment, marked as answer]
I think it is related to issue #83. Currently no fix for this. Dexed and MicroDexed don't have this problem 😟

**Reply by probonopd** [reply]
Fixed in https://github.com/probonopd/MiniDexed/commit/aa5a7c7450a86b95f4eb80e8122a802eefd4accc.
Thanks @arsamus! :+1:

**Comment by Banana71** [comment]
Hi Micheal,
you can hear the sound as soon as you turn the modwheel up a bit.

**Reply by Banana71** [reply]
With Desktop Dexed I can reproduce the behavior by activating "Wheel - EG BIAS" under Parameter (PARM) and turning the ModWheel (Hardware) to "0"

**Reply by thatmichaelpark** [reply]
Thanks for the mod wheel tip!

**Comment by miotislucifugis** [comment]
Not certian, but i suspect that the "quiet" patches are a result of the missing mod sources and destinations in MiniDexed.   The DX7 has 4 modulation sources : mod wheel(cc1), aftertouch, breath(cc2), and foot pedal control(cc3); and 3 destinations: Pitch, amp, and EG Bias.  Currently Minidexed only has mod wheel which is by default set to modulate pitch.     Perhaps these patches have the EG Bias turned all the way down.   @Banana71's   findings seem to support this.
 All of these appear to be supported by synth_dexed.... just havent been added to miniDexed yet.

**Reply by arsamus** [reply]
>With Desktop Dexed I can reproduce the behavior by activating "Wheel - EG BIAS" under Parameter (PARM) and turning the ModWheel (Hardware) to "0".


Yes. I think it isn't a bug. Wheel EG BIAS seems to be set on by default on Minidexed, this could de reproduced on Desktop Dexed as @Banana71 said... That why when A Mod Sense is set to max, no sound can be heared because mod wheel needs to have a value grater than zero. 

The solution seems to be easy, just adding a new menu to set this values. I thing I can do it. This settings needs to be saved on Performances files.

**Reply by probonopd** [reply]
Merged your PR. Thanks @arsamus! :+1:
